.nvmrc

```diff
@@ -0,0 +1 @@
+v12.16.1
```

Comment:
It would be cool to match both: https://github.com/homeday-de/homeday-blocks/blob/develop/.travis.yml#L3